Meta/serenity.sh

```diff
@@ -218,12 +218,16 @@ build_target() {
         # invoked superbuild for serenity target that doesn't set -DBUILD_LAGOM=ON
         cmake -S "$SERENITY_SOURCE_DIR/Meta/Lagom" -B "$BUILD_DIR" -DBUILD_LAGOM=ON
     fi
+
+    # Get either the environement MAKEJOBS or all processors via CMake
+    [ -z "$MAKEJOBS" ] && MAKEJOBS=$(cmake -P "$SERENITY_SOURCE_DIR/Meta/CMake/processor-count.cmake")
```

Comment:
`message` [docs](https://cmake.org/cmake/help/latest/command/message.html):

> (none) or `NOTICE`
> Important message printed to stderr to attract user's attention.

Note that the `processor-count.cmake` uses `(none)`, because that doesn't adorn the message with leading hyphens, so this will never catpure the actual value here. I used `2>&1` for a reason in my previous suggestion.

Reply:
Strange this seemed to work, but I guess empty will fallback on `ninja`s behavior

Reply:
I was worried about other error messages potentially messing up the result, is redirecting it the only good solution?